<launch>
<arg name="robot" default="rosmip" />
<arg name="device" default="/dev/input/js0" />
<group ns="$(arg robot)">
  <!-- Driver for the joystick. -->
  <node pkg="joy" type="joy_node" name="joy" output="screen" >
    <param name="dev" value="$(arg device)" />
    <param name="autorepeat_rate" value="25" />
  </node>

  <!-- MIP specific teleoperation driver -->
  <param name="axis_180_360turn" value="5"/>
  <param name="axis_90turn" value="4"/>
  <param name="axis_angular" value="0"/>
  <param name="axis_linear" value="3"/>
  <param name="button_sound" value="5"/>
  <param name="scale_angular" value="2.5"/>
  <param name="scale_linear" value=".7"/>
  <node pkg="rosmip" type="mip_teleop_joy.exe" name="teleop" output="screen"/>
</group>
</launch>
